src/pages/Login/ResetPassword.js

```diff
@@ -22,14 +21,20 @@ class ResetPassword extends Component {
         });
     }
 
+    passwordLengthError = () => {
+		if (this.state.password.length < 6) {
+			alert("Password is not at least 6 characters");
```

Comment:
I think adding alert, is cheap trick. I suggest to create a popup. And that popup should appear only when you click on reset password button . Currently If I try to click of password input I would get an alert. If you don't want to create popup you can just remove alert.

Reply:
Instead of a popup I would suggest adding some small red text that appears with the error, like in the designs in the forms. And possibly a red border around the input. This is what most forms do